notebooks/entities-synonyms/01_cbow_embeddings.py

```diff
@@ -0,0 +1,58 @@
+from src.utils.epoch_logger import EpochLogger
+import multiprocessing as mp
+from time import time
+
+import pandas as pd
+from gensim.models.phrases import Phrases, Phraser
+from gensim.models import Word2Vec
+
+
+# Reference: https://colab.research.google.com/drive/1A4x2yNS3V1nDZFYoQavpoX7AEQ9Rqtve#scrollTo=m1An-k0q9PMr
+
+n_cores = mp.cpu_count() - 1
+epoch_logger = EpochLogger()
+
+df = pd.read_csv(filepath_or_buffer='data/processed/df.csv',
```

Comment:
I'm not keen on this file being called `df.csv` - it's not clear to someone new to it whats in it. Could it have a more descriptive name of what the df is of?

Reply:
Yup, makes sense, changed it to `content_store_clean.csv` now, https://github.com/alphagov/govuk-entity-personalisation/pull/5/commits/e11cec671aa29a3097db56ed3856275cb128644b.